docs/hyperopt.md

```diff
@@ -375,23 +375,12 @@ Buy hyperspace params:
     'rsi-enabled': True,
     'trigger': 'bb_lower'}
 ROI table:
-{   0: 0.10674752302642071,
-    21: 0.09158372701087236,
-    78: 0.03634636907306948,
+{   0: 0.10674,
+    21: 0.09158,
+    78: 0.03634,
     118: 0}
 ```
 
-This would translate to the following ROI table:
-
-``` python
-minimal_roi = {
```

Comment:
Why would we remove this? 
yes, the table is correct above, too - but the explanation that this is "copy/pasteable" is not available anymore...

Reply:
Because now it's not needed to 'translate' the ROI anymore, it's printed in the format that can simply be copy-pasted into the strategy... (Am I wrong here?)

Reply:
you are right - but it's not mentioned anymore ... 
So "you can use this ROI output and use it as `minimal_roi`" should be added instead i guess

Reply:
But it was not mentioned as 'copy-paste', the text said something about 'translation' (which was mostly rounding of the values), which is no longer needed since the values are already rounded.
I reworked this part (and the stoploss counterpart), pls review.
Also added new subsections for the parts describing default spaces.